apps/chat-e2e/src/tests/promptsNames.test.ts

```diff
@@ -0,0 +1,231 @@
+import dialTest from '@/src/core/dialFixtures';
+import {
+  ExpectedConstants,
+  ExpectedMessages,
+  MenuOptions,
+} from '@/src/testData';
+import { Colors } from '@/src/ui/domData';
+import { expect } from '@playwright/test';
+
+dialTest(
+  'Error message appears if to add a dot to the end of prompt name.\n' +
+    'Prompt name: allowed special characters.\n' +
+    'Prompt name: restricted special characters are not allowed to be entered while renaming.\n' +
+    'Prompt name: restricted special characters are removed from prompt name if to copy-paste.\n' +
+    'Prompt name: smiles, hieroglyph, specific letters in name.\n' +
+    'Prompt name: spaces in the middle of prompt name stay',
+  async ({
+    dialHomePage,
+    promptData,
+    dataInjector,
+    prompts,
+    promptDropdownMenu,
+    promptModalDialog,
+    errorToast,
+    errorToastAssertion,
+    promptAssertion,
+    setTestIds,
+  }) => {
+    setTestIds(
+      'EPMRTC-2991',
+      'EPMRTC-1278',
+      'EPMRTC-2993',
+      'EPMRTC-2994',
+      'EPMRTC-2997',
+      'EPMRTC-3085',
+    );
+    const prompt = promptData.prepareDefaultPrompt();
+    await dataInjector.createPrompts([prompt]);
+    const newNameWithDot = `${ExpectedConstants.newPromptTitle(1)}.`;
+    const nameWithRestrictedChars = `Prompt${ExpectedConstants.restrictedNameChars}_name`;
+    const expectedPromptName = 'Prompt_name';
+    const longNameWithEmojis =
+      '😂👍🥳 😷 🤧 🤠 🥴😇 😈 ⭐あおㅁㄹñ¿äß😂👍🥳 😷 🤧 🤠 🥴😇 😈 ⭐あおㅁㄹñ¿äß😂👍🥳 😷 🤧 🤠 🥴😇 😈 ⭐あおㅁㄹñ¿äß';
+    const nameWithSpaces = ' Prompt 1 ';
+    const expectedNameWithSpaces = 'Prompt 1';
```

Comment:
can be replaced with:
`nameWithSpaces.trim()`

Reply:
done